tls1prf.go

```diff
@@ -102,3 +105,52 @@ func TLS1PRF(result, secret, label, seed []byte, h func() hash.Hash) error {
 	}
 	return nil
 }
+
+// fetchTLS1PRF fetches the TLS1-PRF KDF algorithm.
+// It is safe to call this function concurrently.
+// The returned EVP_KDF_PTR shouldn't be freed.
+var fetchTLS1PRF = sync.OnceValues(func() (C.GO_EVP_KDF_PTR, error) {
```

Comment:
Should this be called `fetchTLS1PRF3` and/or should the comment mention OpenSSL 3?